src/Facade/Services/PurchaseOrderDeliveryFacadeService.cs

```diff
@@ -135,11 +135,66 @@ public class PurchaseOrderDeliveryFacadeService : IPurchaseOrderDeliveryFacadeSe
             }
         }
 
+        public IResult<IEnumerable<PurchaseOrderDeliveryResource>> UpdateDeliveriesForDetail(
+            int orderNumber, 
+            int orderLine,
+            IEnumerable<PurchaseOrderDeliveryResource> resource,
+            IEnumerable<string> privileges)
+        {
+            var purchaseOrderDeliveryResources = resource.ToList();
+            this.domainService.UpdateDeliveriesForOrderLine(
+                orderNumber,
+                orderLine,
+                purchaseOrderDeliveryResources.Select(d => new PurchaseOrderDelivery
```

Comment:
Could this use the buildEntityFromResource method you've got or is it different?